### DISCUSSION TITLE
Chrome插件无法翻译

### DISCUSSION THREAD

**Discussion by tyl0622** [nextai-translator/nextai-translator#697] [Q&A]
<img width="669" alt="image" src="https://github.com/yetone/openai-translator/assets/76834004/ff35d3fd-5654-42b5-8dc1-1a26a9932708">

如图, 使用浏览器插件翻译报错, 查询React错误为:
**Objects are not valid as a React child (found: object with keys {message, type, param, code}). If you meant to render a collection of children, use an array instead.**

插件版本: 0.1.15

**Comment by HaHa-LC** [comment]
我也是报这个错误，service选择的是web服务

**Comment by Muscleape** [comment]
同样是web模式不可用，提示
"SyntaxError: Unexpected token"
web模式下也没有可以输入token的地方啊
报错截图：
<img width="617" alt="image" src="https://github.com/openai-translator/openai-translator/assets/16682039/29959ca9-bd12-48cc-9d82-19c06dca653d">
配置截图
<img width="411" alt="image" src="https://github.com/openai-translator/openai-translator/assets/16682039/97c5690a-62b4-4ff5-9c28-2101bdab9166">
